SU-SWS/sulgryphon-nextjs
# SUL23-135 | @jdwjdwjdw | Add global typography styles

# READY FOR REVIEW

# Summary
- [SUL23-135](https://stanfordits.atlassian.net/browse/SUL23-135): FE-DEV | style Typography for SUL theme
- Add global styles for headings, update tailwind config
- Manually add section header styles in relevant files
- Update link styles where relevant
- Update caption text where relevant

# Review By (Date)
- When convenient

# Steps to Test
1. Checkout branch or view preview at https://deploy-preview-60--stanford-library.netlify.app/sws-test-314-typography
2. Build out elements noted in [the ticket](https://stanfordits.atlassian.net/browse/SUL23-135), and confirm they look correct, or view [preview](https://deploy-preview-60--stanford-library.netlify.app/sws-test-314-typography)
3. Review code 🌴 

# Associated Issues and/or People
- [SUL23-135](https://stanfordits.atlassian.net/browse/SUL23-135): FE-DEV | style Typography for SUL theme


[SUL23-135]: https://stanfordits.atlassian.net/browse/SUL23-135?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
